.github/workflows/ci.yml

```diff
@@ -105,6 +105,15 @@ jobs:
         uses: nrwl/nx-set-shas@v4
       - name: Install dependencies
         run: npm ci
+      - name: Set custom Chrome path for Windows only
+        if: matrix.os == 'windows-latest'
+        # This path is considered in `testing/setup/src/lib/chrome-path-setup.ts` and used in different test configurations
+        run: |
+          echo "CUSTOM_CHROME_PATH=C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe" | Out-File -FilePath $env:GITHUB_ENV -Append -Encoding utf8
+        shell: pwsh
+      #- name: Log all environment variables
+      #  run: |
+      #      printenv
```

Comment:
🧹 This seems like a leftover.

Reply:
I kept it for the next one that needs to debug it. It is also mentioned in the docs.